### DISCUSSION TITLE
IndexError: boolean index did not match indexed array along dimension 0; dimension is 100 but corresponding boolean dimension is 3

### DISCUSSION THREAD

**Discussion by keli559** [Parcels-code/Parcels#1246] [Q&A]
Trying to run a 3D tracking.
The rest is following the given example:
https://nbviewer.org/github/OceanParcels/parcels/blob/master/parcels/examples/tutorial_nemo_3D.ipynb

Struggling on this line:
`pset = ParticleSet.from_line(fieldset=fieldset, pclass=JITParticle,`
`size=Npart, start=(15, -26.5), finish=(13, -26.5), depth=10.0`
                             `  , time=np.linspace(0.0, dT, Npart))`
                             `#, time=np.ones(100)*3600*25)`
when I set `time=np.ones(100)*3600*25`, it passes, and gives result

However, if I use ` time=np.linspace(0.0, dT, Npart)', it will report the error:

`IndexError: boolean index did not match indexed array along dimension 0; dimension is 100 but corresponding boolean dimension is 3`

I double checked the Npart is set to be 10... There is clearly a mismatch of arrays, but what could have been happening ? Thanks!

It seemed to me that there is a glitch if you send the starting time to be sequential e.g. `np.linspace(0.0, dT, Npart)`, versus the time to be the same among all particles `time=np.ones(100)*3600*25)`.

**Comment by VeckoTheGecko** [comment]
> However, if I use `time=np.ones(100)360025)', it will report the error:

I'm not sure what you mean, that isn't valid Python code. Feel free to edit your question to clarify.

The `time` variable must contain an array of times that the particles are released. This array must be the same length as the number of particles `Npart`.

**Reply by keli559** [reply]
@VeckoTheGecko, take a look at my post now... I just updated it. Thanks!
